Pipfile

```diff
@@ -9,7 +9,8 @@ pytest-mock = "~=3.8"
 boto3 = "~=1.24"
 
 [packages]
-crhelper = "~=2.0"
+synapseclient = "~=2.7"
+urllib3 = '<2.0'
```

Comment:
i don't see urllib3 being used?

Reply:
its a transitive dependency of synapseclient, and for some reason I was getting an install error with 2.7.1, but it seems to be resolved now with 2.7.2 so I can remove this pin